docs/guides/execution-modes.mdx

```diff
@@ -13,71 +13,63 @@ The `mode` option requires `qiskit-ibm-runtime` 0.24 or later.
 
 <span id="failed-job"></span>
-## Failed and canceled jobs
+### Failed and canceled jobs
 
 When a job is failed or canceled, the reported usage is as follows:
 
 * Job or batch mode: the reported usage is the time the QPU was locked for executing your workload until the time it failed or was canceled. Therefore, if the failure or cancellation occurred before the lock, the reported usage is zero. Otherwise, the workload's reported usage is the value that Qiskit Runtime returns as `consumed`. Thus, some failed jobs do not appear in your reported usage and others do.
 
 * Session mode: the reported usage is the wall-clock time from the when the first job started executing in the session until the session terminates, regardless of the number of jobs that fail or are canceled.
 
-<span id="ittl-by-mode"></span>
-## Interactive time to live (ITTL)
+<span id="best-practices"></span>
+## Prepare to use execution modes
+
+To ensure the most efficient use of the execution modes, the following practices are recommended:
 
-The [ITTL](/guides/max-execution-time#session-maximum-execution-time) in batch mode is one second for all instances.
+- There is a fixed overhead associated with running a job. In general, if each of your jobs uses less than one minute of QPU time, consider combining several into one larger job (this applies to all execution modes). "QPU time" refers to time spent by the QPU complex to process your job.
 
-The ITTL for a session (Premium Plan users only) is 60 seconds, unless there is an instance-specific configuration that dictates a different ITTL.
+    A job's QPU time is listed in the **Usage** column on the IBM Quantum Platform [Workloads](https://quantum.ibm.com/workloads) page, or you can  query it by running this command: `job.metrics()["usage"]["quantum_seconds"]` in `qiskit-ibm-runtime`.
 
+- If each of your jobs consumes more than one minute of QPU time, or if combining jobs is not practical, you can still run multiple jobs in parallel. Every job goes through both classical and quantum processing. While a QPU can process only one job at a time, up to five classical jobs can be processed in parallel. You can take advantage of this by submitting multiple jobs in [batch](run-jobs-batch#partition) or [session](run-jobs-session#two-vqe) execution mode.
 
-| Execution mode | Instance type (Open or Premium Plan) | ITTL |
-| --- | --- | --- |
-| Batch mode | Both Open and Premium Plan | 1 sec |
-| Session mode | Premium Plan | 60 sec* |
+The above are general guidelines, and you should tune your workload to find the optimal ratio, especially when using sessions. For example, if you are using a session to get exclusive access to a backend, consider breaking up large jobs into smaller ones and running them in parallel. This might be more cost effective because it can reduce wall clock time.
```

Comment:
```suggestion
The above are general guidelines, and you should tune your workload to find the optimal ratio, especially when using sessions. For example, if you are using a session to get exclusive access to a backend, consider breaking up large jobs into smaller ones and running them in parallel. This might be more cost-effective because it can reduce wall clock time.
```